src/meson.build

```diff
@@ -127,12 +129,22 @@ gt_library_dep = declare_dependency(
 
 pkg.generate(
   libraries : gt_library,
-  name : 'gnome-twitch',
+  name : 'gnome-twitch-' + api,
   description : 'Library used for building GNOME Twitch plugins',
   requires : ['gtk+-3.0', 'libpeas-1.0'],
   install : true,
   version : meson.project_version())
 
+gnome.generate_gir(gt_library,
+            sources: [meson.current_source_dir()+'/../include/gnome-twitch/gt-player-backend.h'],
```

Comment:
You should never be using `meson.current_source_dir()` you should be able to use a relative path or worst case a `files()` call.